projects/picker/src/lib/date-time/calendar.component.html

```diff
@@ -68,7 +38,44 @@
         </span>
       </span>
     </button>
+    <button
+      [style.visibility]="currentView !== DateView.MONTH || !showTodayButton ? 'hidden' : 'visible'"
```

Comment:
Because of the flex style being space-between, removing the element would re-align the "Nov 2024" header, unless I applied conditional styling or did it some other way.

Reply:
in this case i normally do sth like:

```html
<button>Nov 2024</button>

<div style="flex-grow"></div>

<button>Today</button>
<button>prev</button>
<button>next</button>
```

obviously without inline styles but you get what i mean 😄
Now you can use `@if` because of the div that just fills the space.

Reply:
@NetWin apparently just the `@if` works and `space-between` already falls back to a "start" alignment when there is only one element, instead of centering it, as I would expect. Suspicious to me, but I digress :joy: